libminifi/src/sitetosite/RawSocketProtocol.cpp

```diff
@@ -569,15 +569,15 @@ bool RawSiteToSiteClient::transmitPayload(const std::shared_ptr<core::ProcessCon
 
     int16_t resp = send(transactionID, &packet, nullptr, session);
     if (resp == -1) {
-      throw Exception(SITE2SITE_EXCEPTION, "Send Failed");
+      throw Exception(SITE2SITE_EXCEPTION, "Send Failed in trasaction " + transactionID);
```

Comment:
We lose this transaction ID in the exception. Is this at least logged somewhere?

Reply:
```exception.what()``` is logged in the corresponding catch block responsible for handling this exception, I think we are good with that.

Reply:
And that includes the transaction ID in that log statement?

Reply:
Whi not? It's a string concat here, so it's going to part of "what()". Or do I need more coffee?

Reply:
No I do. Though more coffee is always good for you!

 Sorry -- I pulled this to review on my branch and saw the conflict because I had a similar change . Mine said, "Failure sending via transaction" versus "in trasaction". I quickly glanced at it, so that is my error; however, since there were linter errors and another commit is needed could you kindly fix the typo? Not a big deal.

Reply:
typo being "in trasaction" vs "in transaction"

Reply:
Haha, okay, didn't spot it, thanks! 
Fixed all the 3 occurrences. :)